docs/manual/java/guide/devmode/ServiceLocator.md

```diff
@@ -52,13 +52,64 @@ The above ensures that the Service Locator knows about the `weather` service. Th
 
 Note that if the service you want to communicate with is actually a Lagom service, you may want to read the documentation for [[integrating with an external Lagom projects|MultipleBuilds]].
 
+
+# Service Gateway
```

Comment:
I thought it would get its own page. I know that, internally, the service gateway and service locator are the same service in development mode, but they are conceptually distinct. I don't think that people would know to look in this page for service gateway information.

Reply:
Fair enough. What do you think about just changing the page title in the TOC?

Reply:
👍